specification/resources/resource-manager/Microsoft.AzureCIS/stable/2021-02-01/dstsServiceAccount.json

```diff
@@ -0,0 +1,415 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Foundational ARM Client",
+    "description": "DstsServiceAccount Provider",
+    "version": "2021-02-01"
```

Comment:
Note adding the specs in azure-rest-api-specs repo means the API will be publicly available. Ensure this is what you need.